reference/annotation/acl.rst

```diff
@@ -0,0 +1,96 @@
+@Acl
+====
+
+The ``@Acl`` annotation is used to create a new access control list and to protect the controller
+that is annotation with this ACL accordingly:
+
+.. code-block:: php
+    :linenos:
+
+    /**
+     * @Acl(
+     *     id="user_user_view",
+     *     type="entity",
+     *     class="OroUserBundle:User",
+     *     permission="VIEW"
+     * )
+     */
+    public function demoAction()
+    {
+        // ...
+    }
+
+Options
+-------
+
+``class``
+~~~~~~~~~
+
+**type**: ``string``
+
+When the `type`_ option is set to ``entity``, the fully qualified class name configured with the
+``class`` option is used to decide whether or not the ACL has to be evaluated when checking if a
+user has access to a certain object. Each object ACL is only responsible for protecting access to a
+certain kind of objects.
+
+``group``
+~~~~~~~~~
+
+**type**: ``string``
+
+ACLs can optionally be grouped. A group is identified by its name.
+
+``label``
+~~~~~~~~~
+
+**type**: ``string``
+
+A human-readable label that can be presented to the users.
+
+``permission``
+~~~~~~~~~~~~~~
+
+**type**: ``string``
+
+When the `type`_ is set to ``entity``, access can be granted based on the action that should be
+performed with a domain object. There are four types of permission which can be granted to a user:
+
+``CREATE``
+
+    The user can create new objects of this entity.
```

Comment:
Please add, that access level of this permission uses to limit the list of available owners for new entity.